### DISCUSSION TITLE
fix cypress seed for local based conduit project

### DISCUSSION THREAD

**Discussion by melonwark** [open-tutorials/cypress#65] [Ideas]
В последнем туториале на 9 шаге( поднятие 3 контейнеров через докер-композ) всегда падает последний контейнер с сайпрессом. Это происходит по нескольким причинам, в файле config.js указано ``` rejectUnauthorized: false ```, собственно когда контейнер с сайпрессом пытается выполнить seed, то постгрес ругается что вы указали параметр ``` rejectUnauthorized: false ``` и что это небезопасно, это можно обойти передав вторым аргументом в функци cy.exec ```{failOnNonZeroExit: false}``` , но сам код теста подразумевает что команда exec должна завершиться с 0 кодом(без ошибок) чего собственно не произойдет т.к. постгресс будет и дальше ругаться на ``` rejectUnauthorized: false ```, поэтому из теста exec следует убрать проверку на статус код: 
```
before(() => {
        cy.exec('npx -w backend sequelize-cli db:seed:all',{failOnNonZeroExit: false})
    });
```
Далее в тесте надо изменить урл сайта на который мы заходим
```cy.visit('http://localhost:3000');   =>   cy.visit('/');```
Так же я бы изменил Dockerfile контейнера самого проекта, т.к. там стоит команда на создание бд, собственно если бд уже была создана и вы хотите повторно запустить контейнер, то контейнер не поднимется т.к. упадет с ошибкой ```database conduit already exist exit with non zero-statuc code```, для избежание этого можно отредактировать Dockerfile на последней строчке:
```CMD npx -w backend sequelize-cli db:drop && npx -w backend sequelize-cli db:create && npm run start -w backend```

**Comment by Vladimir-Tarasov** [comment]
Добрый день. Спасибо за замечание. Проверим
